src/main/java/org/prebid/server/auction/BidderAliases.java

```diff
@@ -1,37 +1,64 @@
 package org.prebid.server.auction;
 
 import org.apache.commons.lang3.ObjectUtils;
+import org.prebid.server.bidder.BidderCatalog;
 
 import java.util.Collections;
 import java.util.Map;
+import java.util.Objects;
 
 /**
  * Represents aliases configured for bidders - configuration might come in OpenRTB request but not limited to it.
  */
 public class BidderAliases {
 
-    private Map<String, String> aliasToBidder;
+    private final Map<String, String> aliasToBidder;
 
-    private Map<String, Integer> aliasToVendorId;
+    private final Map<String, Integer> aliasToVendorId;
+
+    private final BidderCatalog bidderCatalog;
+
+    private BidderAliases(
+            Map<String, String> aliasToBidder, Map<String, Integer> aliasToVendorId, BidderCatalog bidderCatalog) {
 
-    private BidderAliases(Map<String, String> aliasToBidder, Map<String, Integer> aliasToVendorId) {
         this.aliasToBidder = ObjectUtils.firstNonNull(aliasToBidder, Collections.emptyMap());
         this.aliasToVendorId = ObjectUtils.firstNonNull(aliasToVendorId, Collections.emptyMap());
+        this.bidderCatalog = Objects.requireNonNull(bidderCatalog);
+    }
+
+    public static BidderAliases of(
+            Map<String, String> aliasToBidder, Map<String, Integer> aliasToVendorId, BidderCatalog bidderCatalog) {
+
+        return new BidderAliases(aliasToBidder, aliasToVendorId, bidderCatalog);
     }
 
-    public static BidderAliases of(Map<String, String> aliasToBidder, Map<String, Integer> aliasToVendorId) {
-        return new BidderAliases(aliasToBidder, aliasToVendorId);
+    public static BidderAliases of(BidderCatalog bidderCatalog) {
+        return new BidderAliases(null, null, bidderCatalog);
     }
 
     public boolean isAliasDefined(String alias) {
-        return aliasToBidder.containsKey(alias);
+        return aliasToBidder.containsKey(alias) || bidderCatalog.isAlias(alias);
     }
 
     public String resolveBidder(String aliasOrBidder) {
-        return aliasToBidder.getOrDefault(aliasOrBidder, aliasOrBidder);
+        return aliasToBidder.containsKey(aliasOrBidder)
+                ? aliasToBidder.get(aliasOrBidder)
+                : ObjectUtils.firstNonNull(resolveAliasViaCatalog(aliasOrBidder), aliasOrBidder);
     }
 
     public Integer resolveAliasVendorId(String alias) {
-        return aliasToVendorId.get(alias);
+        return aliasToVendorId.containsKey(alias)
+                ? aliasToVendorId.get(alias)
+                : resolveAliasVendorIdViaCatalog(alias);
+    }
+
+    private String resolveAliasViaCatalog(String aliasOrBidder) {
```

Comment:
Just to think. Do we really resolving alias here?
m/b it is better to name like `resolveBidderViaCatalog(...)`.
i can speak crazy, but just consider.